src/components/Home/Keshav/Week5/Counter/styles.ts

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+export const CounterWrapper = styled.div`
+  .counterHeading {
+    padding: 10px;
+    background-color: #000000;
+    color: #fff;
+  }
+  .btn {
+    padding: 15px;
+    background-color: #000000;
+    color: #fff;
+    cursor: pointer;
+    &:hover {
+      background-color: #fff;
+      color: #000000;
+      transition-duration: 1s;
+    }
+    &:active {
+      transform: scale(0.9);
+    }
+  }
+`;
```

Comment:
check the color code format within the complete file

Reply:
Done!